lib/api/controllers/funnelController.js

```diff
@@ -487,7 +485,6 @@ class FunnelController {
             return Bluebird.reject(modifiedRequest.error);
           })
           .catch(() => {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove curly braces and "return" from this arrow function body. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS3524)